lib/experimentUtils.ts

```diff
@@ -65,4 +65,37 @@ export function getProposedEndDate (startDate : string | null, proposedDuration
   const formattedDate = jsDate.toISOString().slice(0, 10)
 
   return formattedDate
-}
\ No newline at end of file
+}
+
+/**
+ * 
+ * @param values - the message contents
+ * 
+ * @param localizations - the localization object contained in the 
+ *                        experiment recipe. May be null.
+ */
```

Comment:
Please add a detailed description of what the function is supposed to do.

Since we're adding new code here, can you write at least a few tests for the most important cases?